examples/netlify-functions/functions/addToCart/addToCart.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const { Cart } = require('../../models');
+
+const handler = async(event) => {
+  try {
+    if (event.body.cartId) {
+      // get the document containing the specified cartId
+      const cart = await Cart.findOne({ _id: event.body.cartId });
+      cart.products.push(event.body.product);
+      await cart.save();
+      return { statusCode: 500, body: cart };
```

Comment:
`statusCode: 500` means this ends up as an error on the frontend. Make `statusCode: 200` unless you mean to throw an error.